Modules/Devices/test/Devices.Domain.Tests/TierNameTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Backbone.Modules.Devices.Domain.Aggregates.Tier;
+using FluentAssertions;
+using Xunit;
+
+namespace Devices.Domain.Tests;
+public class TierNameTests
+{
+    [Fact]
+    public void Can_create_tier_name_with_valid_value()
+    {
+        var validLength = 30;
+        var validTierName = TestDataGenerator.GenerateString(validLength);
```

Comment:
What I don't like about this approach is that you have no idea what the string looks like. So maybe we should use a hardcoded value here. In fact, even multiple values:
- "a-tier-name"
- "A Tier Name"